src/components/Book.js

```diff
@@ -13,12 +15,42 @@ function Book({ title, author, id }) {
 
   return (
     <>
-      <div className="books">
-        <h2 className="book-title">{title}</h2>
-        <p className="book-author">{author}</p>
-        <button type="button" className="remove-btn" onClick={deleteBook} value={id}>Remove</button>
+      <div className="book-item">
+        <div>
+          <h2 className="book-title">{title}</h2>
+          <p className="">{author}</p>
+          <div className="links-container">
+            <ul>
+              <li><button type="button" className="btn-1 line">Comments</button></li>
+              <li><button type="button" className="btn-1 line">Edit</button></li>
+              <li><button type="button" className="btn-1" onClick={deleteBook} value={id}>Remove</button></li>
+            </ul>
+          </div>
+        </div>
+        <div className="progress">
+          <CircularProgressbar
+            className="progressBar"
+            variant="determinate"
+            color="primary"
+            sx={{
+              width: 50,
+            }}
+            value={50}
+          />
+          <div className="numerics">
+            <span className="number">50%</span>
+            <span className="completed">completed</span>
+          </div>
+        </div>
+        <div className="view-progress">
+          <span className="chapter-title">Current chapter</span>
+          <h3 className="chapter-number">Chapter 10</h3>
+          <button type="button" className="btn-2">
+            Update progress
+          </button>
+        </div>
       </div>
     </>
   );
-}
+};
```

Comment:
- [ ] The underline should be grey and should not span from one end of the page to another.
### Your implementation
![image](https://user-images.githubusercontent.com/45460192/201422610-8f9de2d1-6777-4c71-9faa-9d1a9455063d.png)

### Required
![image](https://user-images.githubusercontent.com/45460192/201422491-e3bb6f7d-0479-4598-be04-ad83ffaa86ae.png)